pyro/contrib/easyguide/easyguide.py

```diff
@@ -6,7 +6,6 @@
 
 import torch
 from contextlib2 import ExitStack
```

Comment:
I think its probably time we can remove `contextlib2` from setup.py and just use the standard `contextlib` library in python 3?

Reply:
Let's add that as a task for a separate PR.